### DISCUSSION TITLE
Thank you!!

### DISCUSSION THREAD

**Discussion by ChristianCiach** [ph4r5h4d/wait4it#110] [General]
We've searched a while for a tool like this, and this one fits the bill perfectly!

Thank you especially for providing clean images on docker hub, with clear versioning and regular dependency updates (zero CVEs according to Trivy)! Awesome work!

**Comment by ph4r5h4d** [comment]
Hej @ChristianCiach, very weird that I've not seen this message.
Thank you for the feedback, I tend to keep this utility this way and maybe in the near future I'd publish the Linux packages to make it easier to install and use.
